src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -880,7 +898,7 @@
       - name: Update a kubernetes cluster with standard SKU load balancer to use two AKS created IPs for the load balancer outbound connection usage.
         text: az aks update -g MyResourceGroup -n MyManagedCluster --load-balancer-managed-outbound-ip-count 2
       - name: Update a kubernetes cluster with standard SKU load balancer to use the provided public IPs for the load balancer outbound connection usage.
-        text: az aks update -g MyResourceGroup -n MyManagedCluster --load-balancer-outbound-ips <ip-resource-id-1,ip-resource-id-2>
+        text: az aks update -g MyResourceGroup -n MyManagedCluster --load-balancer-outbound-ips `<`ip-resource-id-1,ip-resource-id-2>
```

Comment:
Why is the `<` wrapped in single quotation marks?

Reply:
updated, I think it somehow got added accidently during a merge conflict.

Thanks for the catch!